code/game/turfs/open/floor/fancy_floor.dm

```diff
@@ -122,6 +125,9 @@
 	clawfootstep = FOOTSTEP_SAND
 	heavyfootstep = FOOTSTEP_GENERIC_HEAVY
 
+/turf/open/floor/grass/snow/spawniconchange()
```

Comment:
try to detect the real issue here

Reply:
by yours power combined, i am captain planet